fix bugs on downloads page

NishiPhalke opened the pull request:
Fixing https://github.com/weng-lab/SCREEN2.0/issues/366

jpfisher72 reviewed (changes requested):
Small thing: Gene Links zip gives you home/niship/genelinks, should be just genelinks folder

NishiPhalke commented:
> Small thing: Gene Links zip gives you home/niship/genelinks, should be just genelinks folder

Changed this url. By **default** windows doesn't tar.gz, Hence, changed it to zip file
